CODEOWNERS

```diff
@@ -1,6 +1,5 @@
-* @defenseunicorns/uds-experience @defenseunicorns/uds-cli
+* @defenseunicorns/uds-experience
 
 # Additional privileged files
 /CODEOWNERS @jeff-mccoy @daveworth
-# NOTE: No E to catch LICENSE and LICENSING
```

Comment:
just curious why we are deleting?   We have this all over

Reply:
because its ugly and stupid... imo

Reply:
was just cleaning up /s